dist/test_data.py

```diff
@@ -246,12 +246,12 @@ def __ge__(self, other):
         Config("burst_duration", 90, r'''
         How long the insertions will occur for.''')]),
     'cache_resize' : Method(test_config),
-    'cursor_bound_01' : Method(test_config),
-    'cursor_bound_02' : Method(test_config + [
+    'bounded_cursor_prefix_indices' : Method(test_config),
```

Comment:
np: Can we alphabetize this list?

Reply:
I think this is alphabetized, but I will fix the rest you mentioned. :)

Reply:
[cbad707](https://github.com/wiredtiger/wiredtiger/pull/8223/commits/cbad70778d40291639468372f21fe74e194e0e09)